hack/config-default.sh

```diff
@@ -16,3 +16,6 @@ default_csv_version="${docker_tag/latest/0.0.0}"
 default_csv_version="${default_csv_version/devel/0.0.0}"
 [[ $default_csv_version == v* ]] && default_csv_version="${default_csv_version/v/}"
 csv_version=${CSV_VERSION:-$default_csv_version}
+
+latest_release_tag=${LATEST_RELEASE_TAG:-"v0.17.0"}
```

Comment:
i'd like to be able to automate this... should we be sorting 'git tag' by semver?